.github/workflows/vast.yaml

```diff
@@ -434,44 +468,56 @@ jobs:
           # the merge-base for pull requests.
           docker buildx build . -t tenzir/${{ matrix.docker-vast.name }}:latest --target production --load \
             --platform linux/amd64 \
-            --build-arg "VAST_BUILD_OPTIONS=${docker_build_args}" \
+            --build-arg "VAST_BUILD_OPTIONS=${cmake_version_args}" \
             --label "org.opencontainers.image.source=https://github.com/tenzir/vast" \
             --cache-to "type=registry,mode=max,ref=ghcr.io/tenzir/${{ matrix.docker-vast.name }}:build-cache" \
             --cache-from "type=registry,ref=ghcr.io/tenzir/${{ matrix.docker-vast.name }}:build-cache"
           if [[ "${{ matrix.docker-vast.upload-dev-images }}" == "true" ]]; then
             docker buildx build . -t tenzir/${{ matrix.docker-vast.name }}-dev:latest --target development --load \
               --platform linux/amd64 \
-              --build-arg "VAST_BUILD_OPTIONS=${docker_build_args}" \
+              --build-arg "VAST_BUILD_OPTIONS=${cmake_version_args}" \
               --label "org.opencontainers.image.source=https://github.com/tenzir/vast" \
               --cache-from "type=registry,ref=ghcr.io/tenzir/${{ matrix.docker-vast.name }}:build-cache"
             docker buildx build . -t tenzir/${{ matrix.docker-vast.name }}-deps:latest --target dependencies --load \
               --platform linux/amd64 \
-              --build-arg "VAST_BUILD_OPTIONS=${docker_build_args}" \
+              --build-arg "VAST_BUILD_OPTIONS=${cmake_version_args}" \
               --label "org.opencontainers.image.source=https://github.com/tenzir/vast" \
               --cache-from "type=registry,ref=ghcr.io/tenzir/${{ matrix.docker-vast.name }}:build-cache"
           fi
           if [[ "$GITHUB_EVENT_NAME" == "pull_request" ]]; then
             docker tag tenzir/${{ matrix.docker-vast.name }}:latest "ghcr.io/tenzir/${{ matrix.docker-vast.name }}:${ref_tag}"
             docker push "ghcr.io/tenzir/${{ matrix.docker-vast.name }}:${ref_tag}"
           fi
+      - name: Login to Docker Hub
+        if: github.event_name == 'push' || github.event_name == 'release'
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - name: Configure AWS Credentials
+        # if: contains(matrix.docker-vast.registries, '622024652768.dkr.ecr.eu-west-1.amazonaws.com')
+        if: matrix.docker-vast.name == 'vast-manager'
+        uses: aws-actions/configure-aws-credentials@v1-node16
```

Comment:
any reason why we are not using `aws-actions/configure-aws-credentials@v2` here?

Reply:
That version did not exist at the time I wrote this.